front_end/Matching/HomeScreen.js

```diff
@@ -1 +1,25 @@
-//Matching-Home
\ No newline at end of file
+//Matching-Home
+import React, {Component} from 'react';
+import {
+    SafeAreaView,
+    Text,
+    View 
+} from 'react-native';
+
+import QuestionList from './QuestionList';
+
+export default class HomeScreen extends Component {
+    constructor(props){
+        super(props);
+        this.state = {
+          questions: props.questions
+        };
+      }
+    render(){
```

Comment:
For the sake of consistency, I think it would be nice to have a line of whitespace between functions

Reply:
Have you guys considered using some kind of auto formatter?